Diagnostics/HealthChecker/Analyzer/Security/Invoke-AnalyzerSecuritySettings.ps1

```diff
@@ -233,7 +233,8 @@ Function Invoke-AnalyzerSecuritySettings {
             -DisplayGroupingKey $keySecuritySettings `
             -DisplayWriteType "Red" `
             -DisplayCustomTabNumber 2
-    } elseif ($null -eq $HealthServerObject.ExchangeInformation.BuildInformation.AffectedByFIPFSUpdateIssue) {
+    } elseif ($null -eq $HealthServerObject.ExchangeInformation.BuildInformation.AffectedByFIPFSUpdateIssue -and
```

Comment:
We should do role checking at one place. We're already checking for `Edge` and `None` server role in `Get-ExchangeInformation` and should move the `ClientAccess` check to this location as well. 
<img width="736" alt="image" src="https://user-images.githubusercontent.com/40993616/149081714-c00bb6d5-c398-4065-b1e5-35e6dac8f05f.png">